src/app/shared/notification/notification.service.ts

```diff
@@ -28,11 +29,31 @@ export class NotificationService {
     this.showToast(message, 'error', true);
   }
 
+  showProblem(problem: ProblemResponse): void {
+    this.showProblemToast(problem, true);
+  }
+
   clear() {
     this.messageService.clear();
   }
 
   private showToast(message: string, type: string, isSticky = false) {
     this.messageService.add({severity: type, summary: type.toUpperCase(), detail: message, sticky: isSticky});
   }
+
+  private showProblemToast(problem: ProblemResponse, isSticky = false) {
+    const { title, detail, type, errors, status, instance } = problem;
```

Comment:
I like it because it makes my eyes dart left-and-right less when reading the object literal below it.
It also shows which properties from the problem object are used in a single line.

```typescript
const { title, detail, type, errors, status, instance } = problem;
this.messageService.add({
  summary: title,
  detail,
  severity: 'error',
  sticky: isSticky,
  data: {
    type,
    errors,
    status,
    instance,
    messageType: 'problem',
  },
});
```
vs.
```typescript
this.messageService.add({
  summary: problem.title,
  detail: problem.detail,
  severity: 'error',
  sticky: isSticky,
  data: {
    type: problem.type,
    errors: problem.errors,
    status: problem.status,
    instance: problem.instance,
    messageType: 'problem',
  },
});
```

Reply:
Fare enough